lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -0,0 +1,295 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import FileUpload from './file_upload';
+import ShowErrors from './errors_list';
+import ShowProgress from './upload_progress';
+import UserInformation from './user_information';
+
+//TODO
+// edx zendesk APIs
+// access token
+// custom fields ids
+// https://openedx.atlassian.net/browse/LEARNER-2736
+//https://openedx.atlassian.net/browse/LEARNER-2735
```

Comment:
@schenedx I have added a comment that in the follow up story I will make hostname, accesstoken and custom field ids configurable.